### PR TITLE
DSG-2114/Hover Icon Is Shifted

### PR DESCRIPTION
**Pull request by g8shen**
# Basics

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Styling and layout changes
- [ ] Code style changes (formatting, local variables)
- [ ] Refactoring (no functional changes)
- [ ] Build or CI related changes
- [ ] Tests
- [ ] Other... Please describe:

## What changes does this PR introduce?

Some icons in text input boxes previously had some padding. This makes them not touch the edges of the input box in previous development like this:
Screenshot 2023-10-24 at 4 45 02 PM

However this was added to the mat-icon class itself so this was causing the hover to stay in the location without the margin. This makes the hover not centered. A fix to preserve the padding but recenter the hover was to delete the old padding and add a simple class around the icon to move it to the right the same amount of px.

## Screenshot of the affected changes

Updated screenshot after change:
Screenshot 2023-10-25 at 1 08 41 PM
Also did a double check of all the input boxes after making the requested changes.

One's mentioned in the Jira:
**In Edit Transaction Page:**
Before:
Screenshot 2023-10-24 at 4 17 47 PM
After:
Screenshot 2023-10-24 at 4 17 05 PM
**In Claims Page:**
Before:
Screenshot 2023-10-24 at 4 15 45 PM
After:
Screenshot 2023-10-24 at 4 35 23 PM

Screenshots across the rest of the app:
login page:
Screenshot 2023-10-24 at 4 37 59 PM
Screenshot 2023-10-24 at 4 40 07 PM
Transaction Config page:
Screenshot 2023-10-24 at 4 40 41 PM
Screenshot 2023-10-24 at 4 41 22 PM
Schema Page:
Screenshot 2023-10-24 at 4 42 52 PM
Screenshot 2023-10-24 at 4 43 04 PM
Dropdown:
Screenshot 2023-10-24 at 5 04 33 PM
Screenshot 2023-10-24 at 5 06 07 PM

## Checklist

- [ ] I have updated the documentation accordingly
- [x] I have confirmed all acceptance criteria defined by the story have been met
- [ ] I have confirmed that Sonar Qube findings for new code are reviewed and fixed, or will be whitelisted

### CODE REVIEW

**Review by derekdiep42 — COMMENTED** [review]
This looks good to me, not sure if there are any other places where icon highlighted on hover.

Can you confirm that the dropdown down arrow is also highlighted correctly? 👀

**Comment by g8shen** [comment]
@derekdiep42 For sure thanks! I have added a screen shot to description of the pr :)

**Review by derekdiep42 — APPROVED** [review]
LGTM! But I'll let @davidyoung77 or @jgnuvalence have the final say

File: libs/shared/ui/nuverial/src/lib/components/text-input/text-input.component.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
     matInput
   />
   <nuverial-button
+    class="nuverial-button-icon"
```
Comment (davidyoung77) [inline libs/shared/ui/nuverial/src/lib/components/text-input/text-input.component.html:21]: Does this work correctly if we add this instead `buttonStyle="icon"`, then we won't need the additional style specificity

This is how the button is intended to be used

**Review by davidyoung77 — APPROVED** [review]

**Pull request merged**